gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Apr 27 14:30:01 CEST 2016
+#Thu Mar 16 16:54:40 CET 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-all.zip
```

Comment:
You didn't push the gradle .jar files. The gradle wrapper should be updated with `./gradlew wrapper --gradle-version 3.4.1 --distribution-type all